recipes-extended/tmux/tmux_3.%.bbappend

```diff
@@ -1,6 +1,6 @@
 FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
 
-CONFFILES += "\
+SRC_URI += "\
```

Comment:
Make this commit a fixup commit for aa47fc39db1c7964cbe494a3c2e762116efa32cb, and then autosquash it before submitting the PR. eg.
```bash
git reset 1608eb60bc3348f2e9f345df481ece32c94ecbad~
git add recipes-extended/tmux/*
git commit --fixup aa47fc39db1c7964cbe494a3c2e762116efa32cb
git rebase -i --autosquash
```

Reply:
Done